plugins.d/alarm-notify.sh

```diff
@@ -1339,6 +1361,69 @@ send_telegram() {
 }
 
 # -----------------------------------------------------------------------------
+# Team sender
+
+send_team() {
+echo "entering in send_team function"
+    local webhook="${1}" channels="${2}" httpcode sent=0 channel color payload
+
+    [ "${SEND_TEAM}" != "YES" ] && return 1
+
+    case "${status}" in
+        WARNING)  icon="⚠️" && color="FFA500";;
+        CRITICAL) icon="🔥" && color="D93F3C";;
+        CLEAR)    icon="💚" && color="65A677";;
+        *)        icon="♡" && color="0076D7";;
+    esac
+
+    for channels in ${channels}
+    do
+        ## More details are available here regarding the payload syntax options : https://docs.microsoft.com/en-us/outlook/actionable-messages/message-card-reference
+        ## Online designer : https://acdesignerbeta.azurewebsites.net/
+        payload="$(cat <<EOF
+        {
+            "@context": "http://schema.org/extensions",
+            "@type": "MessageCard",
+            "themeColor": "${color}",
+            "title": "$icon Alert ${status} from netdata for ${host}",
+            "text": "${host} ${status_message}, ${chart} (_${family}_), *${alarm}*",
+            "potentialAction": [
+            {
+                "@type": "OpenUri",
+                "name": "Netdata",
+                "targets": [
+                { "os": "default", "uri": "http://$(hostname -f):19999" }
+                ]
+            },
+            {
+                "@type": "OpenUri",
+                "name": "Splunk",
+                "targets": [
+                { "os": "default", "uri": "https://your_url.com" }
```

Comment:
Is this actually supposed to be here?  If so, are you certain it shouldn't be templated and configured by the user from `health_alarm_notify.conf`?

Reply:
I would prepare a template for further use. My goal here is to propose template. In the comment you have all the references if you want to adapt it. I hope it's OK for you guys.

Reply:
I think the part for Netdata is fine, I'm talking specifically about the 'Splunk' thing, with the whole 'https://your_url.com' address.  It looks like it's a left over from testing or example code.

Reply:
You're right, we can remove that part. Done in the next commit.
Actually not only test on my side, I'm pushing metrics to Splunk.